common/Makefile

```diff
@@ -22,7 +22,7 @@ clean:
 
 oshinko-cli:
 	- mkdir -p $(clonepath)
-	- cd $(clonepath) && git clone https://github.com/radanalyticsio/oshinko-cli
+	- cd $(clonepath) && git clone -b v0.1.0 https://github.com/radanalyticsio/oshinko-cli
```

Comment:
ok, but a tag is not a branch, is it?
I thought -b was just for branches.
Ok, I guess I'm happy to be wrong here :)  Thanks!